### PR TITLE
Bug 875371 - a sound when locking the device R=prateekjadhwani

### PR DESCRIPTION
**Pull request by prateekjadhwani**
missing sound when locking the screen.

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7832d70d209e8f0206e379886f42304c6f3b6e4d

File: apps/system/lockscreen/js/lockscreen.js [hunk 1]
```diff
@@ -708,6 +714,19 @@
   };
 
   /**
+   *  Sets UnlockSoundEnabled to true or false
+   *  when the screen gets unlocked
+   */
+  LockScreen.prototype.setUnlockSoundEnabled =
+  function lockscreen_setUnlockSoundEnabled(isUnlockSoundEnabled) {
+    if(isUnlockSoundEnabled) {
+      this.unlockSoundEnabled = true;
```
Comment (albertopq) [inline apps/system/lockscreen/js/lockscreen.js]:
```javascript
this.unlockSoundEnabled = !!(isUnlockSoundEnabled);
```

Comment (albertopq) [inline apps/system/lockscreen/js/lockscreen.js]: Actually, do we need a method for this?

Comment (prateekjadhwani) [inline apps/system/lockscreen/js/lockscreen.js]: With ```this.unlockSoundEnabled = !!(isUnlockSoundEnabled);``` we should not need a method anymore. Let me make those changes and test them.

Thanks.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=44c2b116df939afa6743633f31599464c82777b8

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5a499d24cdd43d38b47d8bac1824c9d5a707c388

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0f3b16b464a25638b193e2dc11826e73dcb1e9cc

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=c6ff421a7785c92788244d5cbcb8f9d3fde4c4e9

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f2f06e315ca08a1f5cf9d104e2ff85f7312ab717

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a93e50938c0ce18f764eecc4c850f0dcacdc7898

**Comment by prateekjadhwani** [comment]
Closing this PR, and reopening it in https://github.com/mozilla-b2g/gaia/pull/32693 after the squash.

**Pull request closed without merging**